gradle/libs.versions.toml

```diff
@@ -234,6 +234,7 @@ plugin-japicmp = { module = "me.champeau.gradle:japicmp-gradle-plugin", version
 plugin-downloadTask = { module = "de.undercouch:gradle-download-task", version = "5.6.0" }
 plugin-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version = "6.25.0" }
 plugin-bnd = "biz.aQute.bnd:biz.aQute.bnd.gradle:6.4.0"
+plugin-bndForJava17 = "biz.aQute.bnd:biz.aQute.bnd.gradle:7.1.0"
```

Comment:
Dependabot struggles with having multiple entries in the version catalog for the same artifact. We may want to consider inlining the older version where it is used and keeping only the newer version here so it gets updated by dependabot. We can do this separate from this PR as I think we have a number of other multiple entries that need to be handled.